src/app/components/DetailWrapper/DetailWrapper.tsx

```diff
@@ -18,7 +18,7 @@ export const DetailLabelText = ({
 		className={twMerge(
 			"no-ligatures text-md font-semibold text-theme-secondary-700 dark:text-theme-secondary-500",
 			minWidth === "sm" && "min-w-16",
-			minWidth === "md" && "min-w-36",
+			minWidth === "md" && "min-w-24",
```

Comment:
I'd pass it as a `className` instead of overriding because this will probably break a view in somewhere else